### DISCUSSION TITLE
Adapting a yazi config to Termux

### DISCUSSION THREAD

**Discussion by b9Joker108** [sxyazi/yazi#1823] [1. Q&A]
In Termux, my daily driver and login shell is `zsh` and I have had `yazi` installed for a while, but haven't really used it. I happened upon someone's dotfiles and they had detailed `yazi` configurations and plugins. I have applied that configuration and its plugins to my system. Subsequently, I have become rather enamoured with yazi. For a terminal file browser, it is exceptionally, both visually beautiful, powerful and functional. The configuration I adapted was for a graphical and non-Termux Android environment. So, I need to adapt it to Termux. I have consulted keymap.toml and it contains this configuration and keybinding to open files: 

```lua
52 append_keymap = [
53  { on = [                                                                                                            
54     "e",
55   ], run = "open", desc = "Open the selected files" },
56   { on = [
57     "E",
58   ], run = "open --interactive", desc = "Open the selected files interactively" },
```

But, selecting a file and pressing `e` or `E` isn't currently working. In Termux, the command `termux-open`, opens the selected file in the appropriate default graphical Android application, whether it be: viewer, reader or editor according to media, filetype and file-extension, in most cases. But, I have not yet determined where or how in the `yazi` configurations, I should make this ammendation. Any assistance greatly appreciated.

**Comment by sxyazi** [comment]
I don't have an available Android system for testing, but I think adding `termux-open` for the `open` and `reveal` opener should work:

https://github.com/sxyazi/yazi/blob/176773b3839e7a9ef4f3798c467daddea6e3bbec/yazi-config/preset/yazi.toml#L38-L42

https://github.com/sxyazi/yazi/blob/176773b3839e7a9ef4f3798c467daddea6e3bbec/yazi-config/preset/yazi.toml#L43-L48

See docs at:

- https://yazi-rs.github.io/docs/configuration/yazi#opener
- https://yazi-rs.github.io/docs/configuration/yazi#open

Also, where can I find the full available options for `termux-open` command? Is there a "reveal" operation in Android — that is, opening the GUI file manager and hovering the cursor over the revealed file, similar to macOS's `open -R` and Windows's `explorer /select`?
